config/tsconfig/react-library.json

```diff
@@ -6,6 +6,7 @@
 		"lib": ["DOM", "ESNext"],
 		"module": "ESNext",
 		"target": "ES6",
-		"jsx": "react-jsx"
+		"jsx": "react-jsx",
+		"moduleResolution": "bundler"
```

Comment:
i'm less certain about this change then doing the same to `jobs`, as we are compiling `ui` as a separate package. 

I did this for now, as the default `moduleResolution` setting proved incorrect, and the probably correct setting of `NodeNext` would require us to add `.js` to every import statement, eg turn

```ts
import { Button } from './Button';
```

into

```
import { Button } from './Button.js';
```

I think this would be good ((although we could also specify `.ts` extensions now)[https://devblogs.microsoft.com/typescript/announcing-typescript-5-7/#path-rewriting-for-relative-paths]), but again would be too many changes at once.